apps/website/src/data/network.ts

```diff
@@ -33,442 +32,436 @@ type NetworkItemCamera = NetworkItemCore & {
 
 type NetworkItemAccessPoint = NetworkItemCore & {
   type: "accessPoint";
-  links?: NetworkItem[];
+  links?: NestedNetworkItem[]; // Some Ubiquiti APs have a second ethernet port
+};
+
+type NetworkItemServer = NetworkItemCore & {
+  type: "server";
+  links?: NestedNetworkItem[]; // Allows the Cloud OBS to be at the root of the network
 };
 
-export type NetworkItem =
+export type RootNetworkItem =
   | NetworkItemSwitch
   | NetworkItemCamera
-  | NetworkItemAccessPoint;
+  | NetworkItemAccessPoint
+  | NetworkItemServer;
+export type NestedNetworkItem = RootNetworkItem & {
+  connection: NetworkConnection;
+};
+export type NetworkItem = NestedNetworkItem | RootNetworkItem;
+
+export const isNetworkItem = (item: unknown): item is NetworkItem => {
+  return (
+    !!item &&
+    typeof item === "object" &&
+    "type" in item &&
+    typeof item.type === "string" &&
+    "name" in item &&
+    typeof item.name === "string" &&
+    "model" in item &&
+    typeof item.model === "string"
+  );
+};
+
+export const isRootNetworkItem = (item: unknown): item is RootNetworkItem => {
+  return (
+    isNetworkItem(item) &&
+    (!("connection" in item) || typeof item.connection === "undefined")
+  );
+};
```

Comment:
this is not used as far as i can see. do you wanna keep it for future purposes?

Reply:
Yeah indeed